### PR TITLE
Adding announcements

### PR DESCRIPTION
**Pull request by shyam-lab**
Once announcements will be added, if condition will be changed to TRUE and the announcement will be displayed with the basic <a> tag.

Also removed sentences, "What interests you" "Not sure? View our most popular datasets" based on Shaun's input.  And added header "Announcements" "Data" "Blog" to differentiate between different components in landing page based on Mike R's input.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 48.2.0 (September 04, 2020)
+
+### New Feature
+- Added announecements structure to landing psge
+
+### Enhancements
+- Updates in landing page to distinguish between Announcement, Blog and Data components
+
+
+
```
Comment (remomueller) [inline CHANGELOG.md:10]: Minor: Remove extra spaces from pull request.

File: app/views/external/landing.html.haml [hunk 2]
```diff
@@ -20,12 +20,20 @@
 #top-menu.top-menu= render "layouts/menu"
 
 .page-box
+  -if FALSE
```
Comment (remomueller) [inline app/views/external/landing.html.haml:23]: This only works because Ruby interprets the false as `nil`. To use `true` and `false` use lowercase `false` here. Also, add a space after the initial dash for consistency. `- if false`.

File: app/views/external/landing.html.haml [hunk 3]
```diff
@@ -20,12 +20,20 @@
...
+    .container
+      %h4 Latest announcements
+      %li
+        <a href="https://eristest6.partners.org/nhlbi-continues-its-support-of-the-national-sleep-research-resource/" title="Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG">Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG</a> (September 2, 2020)
+    %br/
+    %br/
```
Comment (remomueller) [inline app/views/external/landing.html.haml:29]: Don't add breaking spaces in HTML, instead use margin bottom on elements you want to have a space after. In this case you can add `.mb-3` on the container class to get the same effect.

```
.container.mb-3
  %h4 ...
  %li
```

File: app/views/external/landing.html.haml [hunk 4]
```diff
@@ -20,12 +20,20 @@
 #top-menu.top-menu= render "layouts/menu"
 
 .page-box
+  -if FALSE
+    .container
+      %h4 Latest announcements
+      %li
+        <a href="https://eristest6.partners.org/nhlbi-continues-its-support-of-the-national-sleep-research-resource/" title="Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG">Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG</a> (September 2, 2020)
```
Comment (remomueller) [inline app/views/external/landing.html.haml:27]: Use ruby `link_to` method here instead of pure HTML.

```
%li
  = link_to "Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG", "https://eristest6.partners.org/nhlbi-continues-its-support-of-the-national-sleep-research-resource", title: "Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG"
  (September 2, 2020)
```

File: config/routes.rb [hunk 5]
```diff
@@ -65,6 +65,7 @@
     end
   end
 
+
```
Comment (remomueller) [inline config/routes.rb:68]: Minor: These white space changes can be removed from this pull request.

**Review by remomueller — CHANGES_REQUESTED** [review]
I left a few comments here. Can you also rebase your changes for this pull request? Thanks!

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,13 @@
+## 48.2.0 (September 04, 2020)
```
Comment (remomueller) [inline CHANGELOG.md:1]: Note: The `version.rb` file also needs to be updated, however the date above, and the change for the version.rb file should be in a separate commit (with no other changes).

File: app/views/external/landing.html.haml (hunk 2 above)
Comment (shyam-lab) [reply app/views/external/landing.html.haml]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (shyam-lab) [reply CHANGELOG.md]: Done

File: app/views/external/landing.html.haml (hunk 3 above)
Comment (shyam-lab) [reply app/views/external/landing.html.haml]: Done

File: app/views/external/landing.html.haml (hunk 4 above)
Comment (shyam-lab) [reply app/views/external/landing.html.haml]: Done

File: config/routes.rb (hunk 5 above)
Comment (shyam-lab) [reply config/routes.rb]: Done

File: CHANGELOG.md (hunk 6 above)
Comment (shyam-lab) [reply CHANGELOG.md]: Done
